src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/CoolerListener.java

```diff
@@ -124,7 +124,6 @@ private boolean consumeJuice(@Nonnull Player p, @Nonnull ItemStack coolerItem, @
                     p.addPotionEffect(effect);
                 }
 
-                p.setSaturation(6F);
```

Comment:
Why remove this?

Reply:
This makes being fed by the cooler consistent with just drinking the juice normally

Reply:
If this PR is to fix an issue, i'd say stay in scope of that issue and then quesiton if the behaviour needs changing separately. The cooler is a different thing and it may well be intended for it to behave differently? IDK and I dont really have an opinion but I do think keep PRs in scope.

Reply:
Agreed